### DISCUSSION TITLE
QTH gridsquare in settings

### DISCUSSION THREAD

**Discussion by geir75** [foldynl/QLog#516] [General]
Is there a way to add more accuracy in gridsquare in settings? By adding more digits? Seems like there is an upper limit to standard 6 digits.

**Comment by foldynl** [comment]
There is a limit of 8 digits (see attached picture). Why do you need more positions?
![image](https://github.com/user-attachments/assets/fda3fbdc-7490-433a-8a4f-c19ae67e454e)

**Reply by geir75** [reply]
Thanks for your answer. When in statistics -> map it shows location in the middle of the grid square. Maybe I'm just picky, but it would be nice to be able to set more precise location like in qrz.

**Reply by foldynl** [reply]
However, you have to distinguish whether it is beautiful to look or useful. I wrote more about it [here](https://github.com/foldynl/QLog/issues/150#issuecomment-1368460670). 8-digi has a resolution of 900m. My opinion is: such a resolution is only necessary in the case of microwaves, where every meter is significant. Is this your case?

**Reply by geir75** [reply]
Thanks for reply. I don't do microwaves. But it's pretty ok how it's now.
